pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "async-firebase"
-version = "3.2.0"
+version = "3.2.1"
```

Comment:
First of all, thank you for contributing to the project. 
As this is not a bugfix, could you please increment the MINOR digit?